scripts/model/blog.js

```diff
@@ -42,27 +54,24 @@
         url: url,
         success: function(data, message, xhr){
           var newTag=xhr.getResponseHeader('eTag');
-          if (newTag !== localStorage['eTag'+name]){
-            localStorage[name] ='';
-            console.log('getting new  blog data');
+          if (newTag !== localStorage['eTag' + name]){
+            localStorage[name] = '';
+            console.log(newTag, localStorage['eTag' + name]);
+            console.log(name, 'getting new  blog data');
```

Comment:
These comments/console logs are really helpful for me as a play with and try to understand your code. I personally like console logs, but some folks (looking at you, Munir) 😉 might ask for the logs to be commented out in the final PR. I like them though as I am new to your code. Very cool. Thank you!